### PR TITLE
Distributor RateGroups followup

### PR DESCRIPTION
**Pull request by dobras**
#### Changelog notes 

```
* Added enums for RateGroups section and some minor improvements.
```

#### Check during review

- [ ] JSON example extended.
  - [ ] New properties are added to the correct place in the JSON.
- [ ] New properties in the table are added to the correct place.
- [ ] Correct formatting:
  - [ ] Spacing is consistent between titles, sections, tables, ...
  - [ ] Correct JSON format - indentation.
- [ ] DateTime properties should always be defined in ISO format.

### CODE REVIEW

File: distributor-api-v1/changelog.md [hunk 1]
```diff
@@ -2,7 +2,5 @@
 
 ## 26.02.2020
 
-* Added documentation for `hotels/getPaymentConfiguration`
-* Deleted old/legacy `payments/getPaymentGateway`
-* [PR](https://github.com/MewsSystems/gitbook-distributor-guide/pull/24)
-
+* Added documentation for `hotels/getPaymentConfiguration`.
+* Deleted old/legacy `payments/getPaymentGateway` [PR](https://github.com/MewsSystems/gitbook-distributor-guide/pull/24).
```
Comment (dmarchuk) [inline distributor-api-v1/changelog.md]: @dobras Why was this changed? There should at least be a dash or something, imagine if the **PR** text wouldn't be a link, then the sentence doesn't make sense.

Comment (dobras) [reply distributor-api-v1/changelog.md]: 1. There were missing `.`
2. The PR should belong to both right? When it was like that it looked like it does not belong to anything. Lonely. Maybe we should put it to both?

File: distributor-api-v1/operations.md [hunk 2]
```diff
@@ -404,7 +404,7 @@ If the hotel does not use any payment gateway, the value is null. If it does, th
 | :--- | :--- | :--- | :--- |
 | `PaymentCardStorageType` | string [PaymentCardStorageType](operations.md#paymentcardstoragetype) | required | Type of the payment card storage used by enterprise. |
 | `IsMerchant` | boolean | required | Whether the gateway is processed through Mews Merchant or not. |
-| `SupportedCreditCardTypes` | string [CreditCardType](operations.md#creditcardtype) | required | The list of supported payment cards, should be used to enhance UX. |
+| `SupportedCreditCardTypes` | string [CreditCardType](operations.md#creditcardtype) | required | Supported payment card, should be used to enhance UX. |
```
Comment (zdrazil) [inline distributor-api-v1/operations.md]:
```suggestion
| `SupportedCreditCardTypes` | array of [CreditCardType](operations.md#creditcardtype) | required | Supported payment cards, should be used to enhance UX. |
```

File: distributor-api-v1/operations.md [hunk 3]
```diff
@@ -642,12 +642,30 @@ Gives availabilities and pricings for given date interval with product prices in
...
+| `SettlementType` | string [SettlementType](operations.md#settlementtype) | required | Determines if system will charge reservation cost automatically or if you'd like employees to manually process payments. |
+| `SettlementAction` | string [SettlementAction](operations.md#settlementaction) | required | Determines how payment will be taken at time of automatic trigger. Valid if settlement is automatic only. |
+| `SettlementTrigger` | string [SettlementTrigger](operations.md#settlementtrigger) | required | Moment that amount is automatically charged, with offset applying to this time (for example, a 'Creation' trigger with no offset will charge the amount when items are created). If settlement is manual, a task will be created at this moment. |
+| `SettlementOffset` | string | required | Start of the interval in [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601#Durations) that gets added before or after selected settlement trigger (for example, '-1 day' will charge the amount 1 day before). |
```
Comment (zdrazil) [inline distributor-api-v1/operations.md]:
```suggestion
| `SettlementOffset` | string | required | Start of the interval in [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601#Durations) which gets added before or after selected settlement trigger (for example, '-1 day' will charge the amount 1 day before). |
```

File: distributor-api-v1/operations.md [hunk 4]
```diff
@@ -642,12 +642,30 @@ Gives availabilities and pricings for given date interval with product prices in
...
+| `SettlementValue` | decimal | required | Percentage of the total extent cost that's charged automatically (for example, a `1.0` settlement value will charge the full cost of extent included below). Value is charged at the time of settlement trigger plus time difference from offset. |
```
Comment (zdrazil) [inline distributor-api-v1/operations.md]:
```suggestion
| `SettlementValue` | number | required | Percentage of the total extent cost which is charged automatically (for example, a `1.0` settlement value will charge the full cost of extent included below). Value is charged at the time of settlement trigger plus time difference from offset. |
```

File: distributor-api-v1/operations.md [hunk 5]
```diff
@@ -642,12 +642,30 @@ Gives availabilities and pricings for given date interval with product prices in
...
-| `SettlementMaximumNights` | nullable int | required | Maximum number of nights that will be charged automatically (only applies to automatic settlements). The rest will be charged manually. |
+| `SettlementType` | string [SettlementType](operations.md#settlementtype) | required | Determines if system will charge reservation cost automatically or if you'd like employees to manually process payments. |
+| `SettlementAction` | string [SettlementAction](operations.md#settlementaction) | required | Determines how payment will be taken at time of automatic trigger. Valid if settlement is automatic only. |
+| `SettlementTrigger` | string [SettlementTrigger](operations.md#settlementtrigger) | required | Moment that amount is automatically charged, with offset applying to this time (for example, a 'Creation' trigger with no offset will charge the amount when items are created). If settlement is manual, a task will be created at this moment. |
```
Comment (zdrazil) [inline distributor-api-v1/operations.md]:
```suggestion
| `SettlementTrigger` | string [SettlementTrigger](operations.md#settlementtrigger) | required | Moment when amount is automatically charged, with offset applying to this time (for example, a 'Creation' trigger with no offset will charge the amount when items are created). If settlement is manual, a task will be created at this moment. |
```

File: distributor-api-v1/changelog.md (hunk 1 above)
Comment (zdrazil) [reply distributor-api-v1/changelog.md]: Other changes seem otherwise ok, so I've merged it. We can decide on the format of the changelog after merging this as 📝
